plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/building/N4JSOutputConfigurationProvider.xtend

```diff
@@ -28,14 +30,11 @@ class N4JSOutputConfigurationProvider implements IOutputConfigurationProvider {
 	 */
 	override getOutputConfigurations() {
 		val outputConfigurations = <OutputConfiguration>newLinkedHashSet
-		val composedGenerators = ComposedGeneratorRegistry.getComposedGenerators();
-		for (IComposedGenerator composedGenerator : composedGenerators) {
-			for (CompilerDescriptor compilerDescriptor : composedGenerator.getCompilerDescriptors()) {
+			for (CompilerDescriptor compilerDescriptor : compositeGenerator.getCompilerDescriptors()) {
```

Comment:
indentation (and maybe add `;` behind statements)

Reply:
done